README.md

```diff
@@ -14,51 +14,45 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 3 >>>
```

Comment:
Looks like these are two different files, one is "Step 3" the other is "Course Start"